### PR TITLE
Create SpaceUser model

### PR DESCRIPTION
**Pull request by cxasper**

### CODE REVIEW

File: spec/factories/user_spaces.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user_space do
+    start_date FFaker::IdentificationESCO.expedition_date
+    finish_date FFaker::IdentificationESCO.expedition_date
+    space nil
```
Comment (moisesnandres) [inline spec/factories/user_spaces.rb]: remove lines with nil

File: app/models/user_space.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# UserSpace
+class UserSpace < ApplicationRecord
+  validates :start_date, presence: true
+  validates :finish_date, presence: true
+  belongs_to :space
```
Comment (moisesnandres) [inline app/models/user_space.rb]: missing has_many in Space model

File: app/models/user_space.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  belongs_to :user
```
Comment (moisesnandres) [inline app/models/user_space.rb]: missing has_many in User model

File: app/models/user_space.rb (hunk 2 above)
Comment (cxasper) [reply app/models/user_space.rb]: has_many :user_spaces ?

**Pull request merged**